.github/workflows/image-scan-analysis.yml

```diff
@@ -276,7 +272,7 @@ jobs:
           reactions: confused 
 
   build_frontend:
-    if: always() && github.event.pull_request.head.repo.full_name == github.repository
+    #if: always() && github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
hmm, can you explain the motivation behind commenting all of these out? did this not work as you were expecting?
Ideally I'd rather we just removed lines instead of pushing configuration with commented out code, assuming that we are confident that we don't need them.

FYI, it should be possible for you to test all of these actions on your fork, that way you don't need to test on the actual dev branch.